packages/@aws-cdk/pipelines/lib/blueprint/wave.ts

```diff
@@ -0,0 +1,94 @@
+import * as cdk from '@aws-cdk/core';
+import { StageDeployment } from './stage-deployment';
+import { Step } from './step';
+
+/**
+ * Construction properties for a `Wave`
+ */
+export interface WaveProps {
+  /**
+   * Additional steps to run before any of the stages in the wave
+   *
+   * @default - No additional steps
+   */
+  readonly pre?: Step[];
+
+  /**
+   * Additional steps to run after all of the stages in the wave
+   *
+   * @default - No additional steps
+   */
+  readonly post?: Step[];
+}
+
+/**
+ * Multiple Stages that are deployed in parallel
```

Comment:
I think we don't need to capitalize these terms

```suggestion
 * Multiple stages that are deployed in parallel
```